src/influxdb.rs

```diff
@@ -1,5 +1,6 @@
 use std::str::from_utf8;
 
+use base64::{engine::general_purpose, Engine as _};
```

Comment:
Right! I did it this way only because it was described like this in the example, but it does not seem to serve any purpose. I therefore applied your suggestion.

Reply:
Right! I did it this way only because it was described like this in the example, but it does not seem to serve any purpose. I therefore applied your suggestion.